packages/near-fast-auth-signer-e2e-tests/mpc-service.ts

```diff
@@ -0,0 +1,153 @@
+import { KeyPair } from '@near-js/crypto';
```

Comment:
I can be wrong, but I think @hcho112 imported all this from the original fast-auth-signer code. No? 

I think it's better to do it to avoid code duplication and make sure they are always in sync, otherwise we have to remember to update this code when the original code changes.